src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -597,21 +597,39 @@ private AdsDataProvider() {
   }
 
   @DataProvider
-  public static Object[][] fliteTagBroken() {
+  public static Object[][] fliteTagBrokenOasis() {
     return new Object[][]{
         {"adtest", "SyntheticTests/FliteTagBrokenWidth",
-         "Invalid width of the flite unit was passed. Make sure you provide width parameter with numeric value."},
+         "Invalid width of the flite unit was passed. Make sure you provide width parameter with numeric value.", "#mw-content-text .error"},
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "#mw-content-text .error" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)